### ISSUE TITLE
Linter Proposal: Disallow shorthands

### ISSUE THREAD

**Issue by jmeinerz** [brigade/scss-lint#796]
I found myself in need to disallow certain shorthands in a project I'm working on, but didn't find that option in the configs. My proposal is that the Shorthand linter would have a config flag similar to the existing `allowed_shorthands`, but for disallowed ones, that should always be flagged when in use. It would look something like:
`disallowed_shorthands: [1, 3, 4]`

If you want to disallow the usage of shorthands of lenghts 1, 3 and 4.

The functionality I am looking for differs fomr `allowed_shorthands` in the sense that I want, for example, shorthands of size 2 to be flagged as bad.

**Comment by sds** [comment]
Hey @jmeinerz, thanks for suggesting ways we can improve the tool.

Is there any issue with having `allowed_shorthands` be the sole defining option here? In your case, for example, if you want to prevent shorthands of length 2, you can specify the following in your configuration:

```yaml
linters:
  Shorthand:
    allowed_shorthands: [1, 3]
```

The linter could then warn on the following:

```scss
p {
  margin: 1px;
  margin: 1px 2px; // Lint
  margin: 1px 2px 3px;
}
```

Your pull request in #797 could then be simplified to not introduce a new rule, but instead add this check. Since the default is to allow all shorthands, I think it's fine to have this be the default behavior going forward.

Let me know if that makes sense, or if there's a nuance here I am missing. Thanks!

**Comment by jmeinerz** [comment]
Alright, I guess that would be fine. I will change the PR later. Thanks for the feedback.

**Comment by jmeinerz** [comment]
@sds I have just updated #797

**Comment by sds** [comment]
Implemented in #797.
